Python/Product/Analysis/Analyzer/DDG.cs

```diff
@@ -213,6 +213,23 @@ internal class DDG : PythonWalker {
 
         public override bool Walk(ExpressionStatement node) {
             _eval.Evaluate(node.Expression);
+
+            if (node.Expression is ExpressionWithAnnotation annoExpr && annoExpr.Annotation != null && annoExpr.Expression is NameExpression ne) {
```

Comment:
No real reason to restrict `annoExpr.Expression is NameExpression` here. If the parser allowed something else, we should go ahead and assign to it normally.